arrivals-example/src/main/scala/com/pagerduty/arrivals/example/ExampleApp.scala

```diff
@@ -43,14 +48,36 @@ object ExampleApp extends App {
   }
 
   val dogServer =
-    Await.result(Http().bindAndHandle(dogRoute, "localhost", 33000), 5.seconds)
+    Await.result(Http().bindAndHandle(dogRoute, "localhost", DogsUpstream.port), 5.seconds)
+
+  implicit val arrivalsCtx = ArrivalsContext("localhost")
+  implicit val authConfig = new ExampleAuthConfig
+
+  val authProxyDirectives = new AuthProxyRoutes(authConfig)
+  val aggregatorDirectives = new AggregatorRoutes(authConfig)
+
+  import com.pagerduty.arrivals.impl.proxy.ProxyRoutes._
+  import authProxyDirectives._
+  import aggregatorDirectives._
+
+  val ExampleComposedResponseFilter = ExampleResponseFilterOne ~> ExampleResponseFilterTwo
+
+  val apiGatewayRoutes =
+    prefixProxyRoute("cats", CatsUpstream, ExampleComposedResponseFilter) ~
+      prefixAuthProxyRoute("dogs", DogsUpstream) ~
+      prefixAggregatorRoute("all", ExampleAggregator)
+
+  // start the API gateway with Arrivals routes
+  val apiGateway =
+    Await.result(Http().bindAndHandle(apiGatewayRoutes, "0.0.0.0", 8080), 5.seconds)
```

Comment:
🤔 We are starting two `Http()` things here, one in `ArrivalsContext` up above and one here. What's the deal?

Can we replace this with:
```
Await.result(arrivalsCtx.http.bindAndHandle(apiGatewayRoutes, "0.0.0.0", 8080), 5.seconds)
```